code/_onclick/adjacent.dm

```diff
@@ -26,36 +26,44 @@
 		* Passing through in this case ignores anything with the throwpass flag, such as tables, racks, and morgue trays.
 */
 /turf/Adjacent(var/atom/neighbor, var/atom/target = null)
-	var/turf/T0 = get_turf(neighbor)
-	if(T0 == src)
-		return 1
-	if(!T0 || T0.z != z)
-		return 0
-	if(get_dist(src,T0) > 1)
-		return 0
+	var/list/turf/Ts = get_turf(neighbor)
```

Comment:
the `/turf` here doesn't do anything as far as I know.

Reply:
Also for people who get confused like me, this proc is nearly identical to the old one, just tabbed in one step and with a `return TRUE` condensed.